content/java/concepts/calendar/terms/computeFields/computeFields.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.computeFields()'
+Description: 'Converts the current millisecond time value time to calendar field values in fields[].'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.computeFields()`** method of the `Calendar` class converts the current millisecond time value `time` to calendar field values in `fields[]`
+
+## Syntax
+
+```pseudo
+calendar.computeFields()
+```
+
+> **Note:** computeFields() method is a non-static method, it is accessible with the `class` object and if we try to access these methods with the `class` name then we will get an error.
+
+## Example
+
+The example below demonstrated the `computeFields()` method. `CalendarComputeFieldExample` class was instantiated and its calendar set to the year `2024`, `computeFields()` is then called on it to sync up with the `calendar` field values with a new time that is set for the `calendar` object.
+
+```java
+import java.util.*;
```

Comment:
The code should be indented with 2 spaces, tabs are not used in docs. Also update all comments to begin with a capital

Reply:
@caupolicandiaz  I fixed the indentation. What do you intend by "update comments"? What comments?

Reply:
The comments within the code should look like this -

// This is correct

instead of 

//this is not the right format